src/shared/labels/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import classNames from 'classnames';
+import { IconContext } from 'react-icons';
+import { RiCloseFill } from 'react-icons/ri';
+
+import { LabelProps } from './type';
+
+const style = 'flex justify-between text-sm border rounded text-black-light focus:outline-none focus:shadow-outline h-10 px-2';
+
+function Label({
```

Comment:
Can the name of function be more specific? The 'label' name is already used in inputs etc